config/locales/forms/users/hi.yml

```diff
@@ -0,0 +1,11 @@
+hi:
+  helpers:
+    label:
+      user:
+        email: "ईमेल"
+        name: "नाम"
+        country: "देश"
+        educational_institute: "शैक्षिक संस्थान"
+        locale: "लोकेल"
```

Comment:
"लोकेल" is very English-y and not many Hindi speaking folks would understand the meaning. Since this deals with the language part, consider using "bhasha".